.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: "An issue or feature request related to the GUI"
+    url: https://github.com/bitcoin-core/gui/issues/new
+    about: The 'bitcoin-core/gui' repository is used exclusively for the development of the GUI. Its master branch is identical in all monotree repositories. Release branches and tags do not exist, so please do not fork that repository unless it is for development reasons.
```

Comment:
Why is the "about" text changed?

Reply:
Before the "about" was: 

> Any report, issue or feature request related to the GUI should be reported at https://github.com/bitcoin-core/gui/issues/

Before this change someone seeking to open a GUI issue but who starts in the main repo would need to manually enter the GUI repo URL to go to the correct repo. After this change that person is automatically redirected to the GUI repo when clicking on the new GUI issue button from within the main repo. So that person would no longer need to be told to go to that URL in the "about". That is the motivation for changing it (I think without commenting on what the "about" is changed to).

Reply:
@michaelfolkson yes old "about" text doesn't make sense after this change because we have created a link for redirection

@MarcoFalke I copied the new "about" from 2nd paragraph in https://github.com/bitcoin/bitcoin/#development-process

Reply:
Changed `about` text and squashed commits to https://github.com/bitcoin/bitcoin/commit/417ba29c0232cb04bab4b6d02ce039d5ff40127b